app/src/main/java/org/wikipedia/dataclient/okhttp/HttpStatusException.java

```diff
@@ -29,10 +32,10 @@ public HttpStatusException(@NonNull Response rsp) {
         }
     }
 
-    public HttpStatusException(@Nullable ServiceError error) {
-        serviceError = error;
-        code = 0;
-        url = "";
+    public HttpStatusException(int code, String url, String message) {
```

Comment:
Could we have annotations on these parameters?

Reply:
Sure!